helpers/remove-unused.sh

```diff
@@ -1,5 +1,15 @@
 # Remove unused files, or files that should not be used
 
+if [ "$SYSTEM_SPARSE" == "" ]; then
+    SYSTEM_SPARSE=$VENDOR_SPARSE
+fi
+
+if [ "$SYSTEM_SPARSE" == "" ]; then
+    echo "SYSTEM_SPARSE and VENDOR_SPARSE are unset!"
+    exit 1
+fi
+
+rm -rf `find $SYSTEM_SPARSE -name *.apk`
```

Comment:
At least on my ArchLinux `find $SYSTEM_SPARSE -name *.apk` errors out with "find: paths must precede expression: ", due to bash resolving the *.
Quotes make it fix: `find $SYSTEM_SPARSE -name "*.apk"`